pkg/ocm/client/client.go

```diff
@@ -206,6 +209,14 @@ func (c *OCMClient) NewShare(ctx context.Context, endpoint string, r *NewShareRe
 		return nil, errors.Wrap(err, "error doing request")
 	}
 	defer resp.Body.Close()
+	respBody, err := io.ReadAll(resp.Body)
+
+	// defer resp.Body.Close()
+	// body, err := io.ReadAll(resp.Body)
+	if err != nil {
+		log.Debug().Msgf("OCM response from %s: %d; could not read body", url, resp.StatusCode)
+	}
+	log.Debug().Msgf("OCM response from %s: %d %s", url, resp.StatusCode, respBody)
 
 	return c.parseNewShareResponse(resp)
```

Comment:
Not sure this is going to work then, after consuming all the body.
You can think of adding the log in the `parseNewShareReponse`function.